client/admin/components/users/index.vue

```diff
@@ -39,6 +39,10 @@
               <td>{{ props.item.firstName }}</td>
               <td>{{ props.item.lastName }}</td>
               <td class="no-wrap">{{ props.item.createdAt | formatDate }}</td>
+              <td>
+                <span v-if="!props.item.lastActive">Never</span>
+                <timeago v-else :datetime="props.item.lastActive"/>
```

Comment:
It would be nice to use `timeago`s name option to rename it to `v(ue)-timeago` because single word names indicate _native_ html elements...